packages/manager/modules/bm-cluster-components/src/cluster/cluster.html

```diff
@@ -0,0 +1,18 @@
+<header class="oui-header" role="banner">
+    <div class="d-flex">
+        <h1 class="oui-header__title" data-ng-bind="$ctrl.displayName"></h1>
+    </div>
+    <h2
+        class="oui-header__description"
+        data-ng-bind="'Cluster description - To retrieved from an API - Hard coded for now'"
+    ></h2>
```

Comment:
```suggestion
    <h2
        class="oui-header__description"
        data-ng-bind="'Cluster description - To retrieved from an API - Hard coded for now'"
    ></h2>
```
Maybe remove it for now and added it only when the information is available